### ISSUE TITLE
Report errors to client

### ISSUE THREAD

**Issue by mininao** [BDE-ESIEE/API-calendar#11]
Such as here : http://is.gd/Y70kVs

**Comment by minijackson** [comment]
I don't think you should do that. If you rise the error to the client, either the error is reported two times, or you will print an incorrect location in the debug info.

**Comment by mininao** [comment]
The client should be informed of an error in order to make a difference between an empty response and an unavailable response, right ?

**Comment by minijackson** [comment]
Sorry, misunderstood the issue

**Comment by mininao** [comment]
Please send the 500 error code along
